go.mod

```diff
@@ -0,0 +1,30 @@
+module github.com/aerospike/tools-common-go
+
+go 1.21
```

Comment:
It is probably ok to be using the latest minor Go version but I believe this version should always be less than or equal to the oldest version of Go of the tools that import the go-tools-common lib. Let's keep that in mind. I could be wrong though. Maybe it only _really_ matters if the we or the dependencies use incompatible apis.

Reply:
This lib uses errors.Join that was introduced in Go 1.20 so these libs need atleast that. I'll change this to 1.20 and if it breaks anything we can remove the errors.Join.